frontend/src/components/event-card/EventDetail.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import Typography from '../typography';
+import Icon from 'antd/lib/icon';
+
+const ListStyled = styled.li`
```

Comment:
Would `StyledList` be a better naming for this?

Reply:
probably